crates/bitcoin/src/utils.rs

```diff
@@ -4,6 +4,15 @@ use sp_std::{prelude::*, vec};
 
 use crate::types::H256Le;
 
+const P2PKH_IN_WEIGHT: u32 = 148 * 4;
```

Comment:
It's not clear what these weights / sizes describe exactly, can you add a comment?